MVC/model.js

```diff
@@ -117,6 +119,26 @@ class Model {
         }
     }
 
+    async incrementArticleVotes(id, increment) {
+
```

Comment:
We could have allowed psql to throw errors here if it's going to, without needing to handle it ourselves.